### DISCUSSION TITLE
Difference between PIMCORE_CLASS_DIRECTORY and PIMCORE_CLASS_DEFINITION_DIRECTORY ?

### DISCUSSION THREAD

**Discussion by elburro1887** [pimcore/pimcore#14724] [Q&A / Support]
What is the difference between 

- PIMCORE_CLASS_DIRECTORY
- PIMCORE_CLASS_DEFINITION_DIRECTORY

?

There is no real documentation, the only info I can find is here:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Deployment_Tools.html#page_Pimcore-Class-Definitions

But it doesn't really go into any details.

**Comment by elburro1887** [comment]
BUMP

**Reply by NiklasBr** [reply]
I propose you follow [Cunningham's Law](https://meta.wikimedia.org/wiki/Cunningham%27s_Law) and create a PR with incorrect PHP comments guessing or "explaining" what [the constants](https://github.com/pimcore/pimcore/blob/10.6/stubs/dynamic-constants.php) are for. I bet you will get an answer quicker that way 😉
